.github/workflows/build.yaml

```diff
@@ -31,25 +31,41 @@ jobs:
             - name: Setup Gradle
               uses: gradle/actions/setup-gradle@v3
               with:
-                gradle-version: 6.9.2
+                gradle-version: 8.9
             
             - name: Build Integration Tests Gradle source code
               run: |
-                gradle -b galasa-inttests-parent/build.gradle publish publishToMavenLocal \
+                set -o pipefail
+                gradle -b galasa-inttests-parent/build.gradle publish publishToMavenLocal --info \
                 -PsourceMaven=https://development.galasa.dev/gh/maven-repo/obr \
                 -PcentralMaven=https://repo.maven.apache.org/maven2/ \
-                -PtargetMaven=${{github.workspace}}/repo
+                -PtargetMaven=${{github.workspace}}/repo 2>&1 | tee gradle-build.log
+              
+            - name: Upload Maven Build Log
+              if: always()
+              uses: actions/upload-artifact@v4
+              with:
+                name: gradle-build-log
+                path: gradle-build.log
             
             - name: Building Integration Tests Maven source code
               run: |
-                  mvn -f galasa-inttests-parent/dev.galasa.inttests.obr/pom.xml deploy \
+                  set -o pipefail
+                  mvn -f galasa-inttests-parent/dev.galasa.inttests.obr/pom.xml -X deploy \
                   -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/obr \
                   -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
                   -Dgalasa.release.repo=file:${{ github.workspace }}/repo \
                   -Dgalasa.skip.deploytestcatalog=true \
                   -Dgalasa.skip.bundletestcatalog=false \
                   --batch-mode --errors --fail-at-end \
-                  --settings ${{github.workspace}}/settings.xml
+                  --settings ${{github.workspace}}/settings.xml 2>&1 | maven-build.log
+            
+            - name: Upload Maven Build Log
+              if: always()
+              uses: actions/upload-artifact@v4
+              with:
+                name: maven-build-log
+                path: maven-build.log

```

Comment:
Can we rename this to "inttests-obr-maven-build-log" so we can differentiate from the above. The first build is to build the source code of the inttests and this is to build the inttests OBR.

Reply:
done